pkg/pmem-csi-driver/nodeserver.go

```diff
@@ -798,6 +803,11 @@ func findMountFlags(flags []string, findIn []string) bool {
 		}
 		found := false
 		for _, fIn := range findIn {
+			if f == "dax=always" && fIn == "dax" {
```

Comment:
There's no need to have two if checks:
```
if f == "dax=always" && fIn == "dax" ||
  f == "dax" && fIn == "dax=always" ||
  f == fIn {
   found = true
   break
}
```

Note that I added the case for "we use dax" and the kernel reports "dax=always". I don't know whether we need it, but it feels safer to have it.